README.md

```diff
@@ -281,6 +307,18 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Not clarified by now.
 - **Notes**: See this announcement from [Kévin Dunglas on his Blog](https://dunglas.fr/2019/09/api-platform-2-5-revamped-admin-new-api-testing-tool-next-js-and-quasar-app-generators-patch-and-json-schema-support-improved-openapi-and-graphql-support/).
 
+#### **QMK**
+
+- **Swag**: Shirt, Keyboard
+- **Requirements**: Interact 4 times with a QMK repo in any of these ways:
+  - Review/suggest significant changes on a PR that's labelled `hacktoberfest`
+  - Make a comment on an Issue/PR that contributes to the conversation that's labelled `hacktoberfest`
+  - Open a PR that documents any new feature that's consistent with our doc style (check out the `needs doc` label)
+  - Open a PR that translates some documentation
+  - Open a PR that fixes a bug in an issue labelled `hacktoberfest`
```

Comment:
Instead of having a list like this, I'd recommend something like "Requirements are listed on their Hacktoberfest site" with an embedded link. It keeps things cleaner. :smile: